github/REVIEWING.md

```diff
@@ -0,0 +1,341 @@
+
+1. Make sure the commit message is readable and to the point.
+   Make sure it clearly details what the commit does.
+   Make sure it fits the content of the commit (the actual contribution);
+   i.e. when you check the changes in the commit, they are well connected to the message;
+   that is they don't mention more or less than the commit contents.
+
+You will submit any issues in the general review message.
+
+## Review Contents
+
+Reviewing contents is done both in the pull request GitHub interface and locally.
+
+In the pull request GitHub interface, go the to the `Files changed` tab where you can see an interactive view of the changes.
+Walk through the changes and check for anything that may not be good / correct:
+
+- hard to read content
+- typos
+- violation of general coding / writing conventions
+  - lines to large
+  - missing newlines at the end of the files
+  - training white spaces
+  - improper indentation
+- violation of project coding / writing conventions
+- general coding / writing mistakes
+- project-specific coding / writing mistakes
+
+Follow the [GitHub review guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and add your request for changes.
+In the GitHub interactive interface in the `Files changed` tab you can request changes either on a single line or on multiple lines, where the `+` icon appears when hovering.
+You can request changes as feedback text or as concrete content update suggestions, as shown in the [GitHub review guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and add your request for changes.
+
+Locally you can get a unified view of the changes inside files.
+You can open a file and check out its entirety: existing content and newly contributed content.
+You can check the newly contributed content and make sure that:
+
+- It is consistent with existing content: same structure, same format, same conventions.
+- It is readable and integrates well with existing content.
+
+Finally, you want to be sure that the content is well aligned with the commits it is part of.
+Each commit should do one thing and do one thing well, and the content and commit should be aligned.
+
+For that, go through each commit in the contribution, from the oldest to the most recent one.
+Find the commit IDs by using:
+
+```console
+git log
+```
+
+Go to the history and get the oldest commit ID (the commit hash) and then check it.
+Then get the next commit ID and check it.
+And so on.
+
+To check each commit use:
+
+```console
+git show <commit_id>
+```
+
+where `<commit_id>` is the ID (hash) of the commit.
+
+When checking the commit be sure that:
+
+- The commit message and the commit contents correspond.
+  The commit message should not state anything more or less than the contents.
```

Comment:
What about "The commit messages describes the contents entirely and nothing more than the contents"? The "more or less" part makes this confusing.